oxd-server/src/main/resources/oxd-server.yml

```diff
@@ -67,6 +67,7 @@ logging:
 defaultSiteConfig:
   op_host: ''
   op_discovery_path: ''
+  op_configuration_endpoint: ''
```

Comment:
We don't need default for `op_host` and `op_discovery_path` since we deprecate it.

Reply:
Done